### ISSUE TITLE
Create Extension ERROR

### ISSUE THREAD

**Issue by yosuke-nakamura** [laurenz/oracle_fdw#592]
Hi,

When execute "CREATE EXTENSION oracle_fdw" , this message is displayed 
"Extension 'oracle_fdw' has no install script or update path for version '1.2'"

How can I solve this?

Thank you.

**Comment by laurenz** [comment]
That means that you didn't install oracle_fdw correctly. In particular, `oracle_fdw--1.2.sql` is missing.

How exactly did you install the extension?

The file should be in your extension directory, that is the subdirectory `extension` in the result of `pg_config --sharedir`.

**Comment by yosuke-nakamura** [comment]
Thank you for your reply.

Inside the `extension` are the following files.
- oracle_fdw--1.0--1.1.sql
- oracle_fdw--1.1--1.2.sql
- oracle_fdw--1.1.sql
- oracle_fdw--1.2.sql
- oracle_fdw.control

Is it possible that the directory cannot be referenced?

**Comment by laurenz** [comment]
That looks fine. One possibility is that you have several versions of PostgreSQL (say, different versions) on your machine, and you are looking at the wrong `pg_config`. Please double-check that.

The error message suggests that `oracle_fdw.control` could be found, because that's the source of the information that version 1.2 is to be installed. So it looks like a partial installation.

What *I* would do is to `strace` the PostgreSQL backend process , then run `CREATE EXTENSION oracle_fdw;`. That should show what file PostgreSQL wants to access and what problems it encounters; something like

```none
openat(AT_FDCWD, "/usr/pgsql-15/share/extension/oracle_fdw--1.2.sql", O_RDONLY) = -1 ENOENT (No such file or directory)
```

That should help with finding the cause of the problem.

**Comment by yosuke-nakamura** [comment]
Thank you for your reply.
Resolved.

The Postgresql version is 14, but 9 was installed in the development tools.
`rpm -e postgresql-devel-9.2.24-8.el7_9.x86_64`
to uninstall it.

I checked the configuration with `/usr/pgsql-14/bin/pg_config --sharedir`
/usr/pgsql-14/share
and copied oracle_fdw--1.2.sql to the extension directory under it.

Also, I checked the configuration with `/usr/pgsql-14/bin/pg_config --libdir`
/usr/pgsql-14/lib
was displayed, so we copied oracle_fdw.so to that directory.

That solved the problem.

Thank you very much.
